### ISSUE TITLE
Issue with Owner

### ISSUE THREAD

**Issue by Incrested** [cyckl/Luna#12]
Hey, @d-nni3l!

Just a side note, @d-nni3l doesn't know how to code. 11 of his 15 repositories on GitHub (see https://github.com/d-nni3l) are forks, meaning he copied the code from others. One of which is my own (https://github.com/d-nni3l/miku-chan was copied from https://github.com/incrested/miku-chan).

His group, Aurora Software on GitHub (view at https://github.com/auroraware) has one project, which I can't hate on but it's "bolt" software has nothing in it but a license, readme, and a Jekyll configuration file (_config.yml). This group is also on GitLab (view at https://gitlab.com/auroraware), however, it is empty. If this seems like a surprise, it's the same on Cyanide Security (https://gitlab.com/cyanidesec). His commit history (view here: https://gitlab.com/d-nni3l#activity) is full of updates to Markdown files (*.md). Markdown, if you don't know, is basically flavored text files. You can see examples of them with "README.MD", "LICENSE.MD", etc.

The group, Code at MMS (https://gitlab.com/codeatmms) has only one project (which is the one I'm posting this very important issue). Is that really a use of coding?

Overall, Daniel is a fraud who says he can code. He steals other codes on sites and makes groups and hosts files on GitHub and GitLab just for websites. Before this issue gets shut down or broken, I would recommend you think twice about your thought of Daniel and if you still want to keep this kind of thing going where he treats everyone as a script kiddie.

(P.S. A script kiddie is an uneducated twat when it comes to coding and uses scripts/programs from other developers to deface, recreate, and attack other servers, computer systems, projects, programs, and networks.)

**Comment by d-nni3l** [comment]
hey @Incrested! You are absolutely correct about the fact that I can't code whatsoever. I tried learning to a couple months back but ultimately I just abandoned it as it wasn't really a good fit for me. I work more on the design side of things now and it's actually proven to be the right fit for me. Essentially all of my repositories are forks or just empty, and I should probably get around to deleting them (or not idk). Luna was a little project that we did for English last year and I just keep it up so we can look back on it. All the forks and empty proects were really just either a "save for later" type of thing or attempts at doing something with development. On the other hand, code@mms isn't my club. I don't officially run the club on paper, I just help maintain some resources for it and keep people managed. Since I do fully respect your right to posting opinions/facts (depending on the specific subject), I'll keep these issues up just so people can see them. If you have any more concerns feel free to add on or just contact me.
